adapters/Makefile

```diff
@@ -27,3 +29,10 @@ prepare: generate
 
 run: generate build prepare
 		/bin/bash ./script.sh run $(TARGET)
+
+write_results:
```

Comment:
Does this target depend on another? As I understood, it requires some data to generate this report. Then we need a target for this data which depends on `run`: `write_results -> benchmark_data -> run`, where `benchmark_data` can process result data produced by `run` target